src/app/legacy/containers/Navigation/index.canonical.jsx

```diff
@@ -48,8 +60,10 @@ const CanonicalNavigationContainer = ({
             {scrollableListItems}
           </ScrollableNavigation>
         )}
+        <Divider />
       </ScrollableWrapper>
       <CanonicalDropdown isOpen={isOpen}>{dropdownListItems}</CanonicalDropdown>
+      <ScrollablePromo blocks={blocks} experimentVariant={experimentVariant} />
```

Comment:
Similar here, can this be conditionally rendered if `experimentVariant` is a valid value?

Reply:
The conditional rendering was happening in https://github.com/bbc/simorgh/blob/5d4fde832edcb126bbe84e95ee19363986e8028f/src/app/legacy/containers/Navigation/index.jsx. 
Like you say, it can instead be done in Navigation/index.canonical.jsx as we do not want it on amp, so then we could also remove the !isAmp. I will shift the whole thing into this file instead.

Reply:
Moved to canonical file